File: src/main/java/io/github/kingg22/api_vacunas_panama/service/ISedeService.java
package io.github.kingg22.api_vacunas_panama.service;

import io.github.kingg22.api_vacunas_panama.web.dto.UUIDNombreDto;

import java.util.List;

public interface ISedeService {
    List<UUIDNombreDto> getIdNombreSedes();
}

File: src/main/kotlin/io/github/kingg22/api/vacunas/panama/modules/sede/service/ISedeService.kt
package io.github.kingg22.api.vacunas.panama.modules.sede.service

import io.github.kingg22.api.vacunas.panama.modules.common.dto.UUIDNombreDto
import io.github.kingg22.api.vacunas.panama.modules.sede.entity.Sede
import java.util.Optional
import java.util.UUID

interface ISedeService {
    val idNombreSedes: List<UUIDNombreDto>

    fun getSedeById(id: UUID): Optional<Sede>
}
